landmarkerio/template.py

```diff
@@ -68,13 +109,39 @@ def build_json(groups, n_dims):
     return lm_json
 
 
-def load_template(path, n_dims):
+def load_legacy_template(path, n_dims):
     with open(path) as f:
         ta = f.read().strip().split('\n\n')
     groups = [parse_group(g) for g in ta]
     return build_json(groups, n_dims)
 
 
+def group_to_dict(g):
+    data = {'label': g.label, 'points': g.n}
+    if g.index:
+        data['connectivity'] = ['{} {}'.format(c[0], c[1]) for c in g.index]
+    return data
+
+
+def convert_legacy_template(path):
+    with open(path) as f:
+        ta = f.read().strip().split('\n\n')
+    groups = [parse_group(g) for g in ta]
+    data = {'groups': [group_to_dict(g) for g in groups]}
+
+    new_path = path[:-3] + 'yml'
+    if p.isfile(new_path):
```

Comment:
maybe highlight these cases to user? E.g:
```
print('warning - upgrading txt template at path {} collides with existing yaml template at path {}.'
      "appending '-converted' to file name to avoid collision".format(path, new_path))